src/mds/agg_mds/datastore/elasticsearch_dao.py

```diff
@@ -0,0 +1,234 @@
+from elasticsearch import Elasticsearch
+from typing import List, Dict
+from typing import Any
+import json
+from mds import logger
+
+
+agg_mds_index = "commons-index"
```

Comment:
Should we parameterize this to facilitate any future blue-green deployment?

Reply:
I think we can keep it static for now and parameterize when we see a demand/need